### PR TITLE
PAYARA-1379

### PR DESCRIPTION
**Pull request by mulderbaba**
-Services-that-implement-ConfigListener-should-return-null-on-the-configurations-that-they-are-interested

### CODE REVIEW

**Comment by mulderbaba** [comment]
jenkins test please

**Comment by mulderbaba** [comment]
jenkins test please

File: nucleus/payara-modules/healthcheck-core/src/main/java/fish/payara/nucleus/healthcheck/HealthCheckService.java [hunk 1]
```diff
@@ -265,8 +265,11 @@ public UnprocessedChangeEvents changed(PropertyChangeEvent[] events) {
             public <T extends ConfigBeanProxy> NotProcessed changed(TYPE type, Class<T> changedType, T changedInstance) {
                 if(changedType.equals(HealthCheckServiceConfiguration.class)) {
                     configuration = (HealthCheckServiceConfiguration) changedInstance;
+                    return null;
+                }
+                else {
+                    return new NotProcessed("Unimplemented by HealthCheckService");
```
Comment (smillidge) [inline nucleus/payara-modules/healthcheck-core/src/main/java/fish/payara/nucleus/healthcheck/HealthCheckService.java]: I think you should only return a NotProcessed only when you have Vetoed a change. Not when you have ignored a change.

**Comment by mulderbaba** [comment]
jenkins test please

**Comment by smillidge** [comment]
jenkins test please

**Comment by smillidge** [comment]
jenkins test please

**Comment by smillidge** [comment]
jenkins test please

**Pull request merged**